fdk/__init__.py

```diff
@@ -16,15 +16,24 @@
 import os
 import uvloop
 
+from fdk import constants
 from fdk import runner
+from fdk import http_stream
 
 
 def handle(handle_func):
-    with open("/dev/stdin", "rb", buffering=0) as stdin:
-        asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
-        loop = asyncio.get_event_loop()
-        while True:
-            f = runner.handle_request(
-                handle_func, stdin, os.environ.get("FN_FORMAT"))
-            response = loop.run_until_complete(f)
-            response.dump()
+    asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
+    loop = asyncio.get_event_loop()
+
+    format_def = os.environ.get("FN_FORMAT")
+    lsnr = os.environ.get("FN_LISTENER", "unix:/iofs/lsnr.sock")
```

Comment:
There's no default value for FN_LISTENER. If FN_FORMAT is http-stream, then FN_LISTENER has to be set, and it has prefixed by "unix:" by contract, otherwise FDK should just error/panic.